prisma/schema.prisma

```diff
@@ -23,6 +23,74 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
+// Achhievement
+// A final achievement can consist of several under achievements
+model achievement_template {
+  id                            Int                       @id @default(autoincrement())
+  metrics                       String[]                  @default([])
+  templateFor                   template_for_enum
+  // Groups defines the "final-achievement"
+  group                         String                    @unique @db.VarChar
+  // Group order tells in which place a subordinate step / achievement is
+  groupOrder                    Int                      
+  // the final achievement is marked with isLast 
+  isLast                        Boolean
```

Comment:
(argh, if Prisma would support window functions - https://github.com/prisma/prisma/issues/7039, we could just do `LAST_VALUE(groupOrder) = groupOrder AS isLast OVER group` ... But I guess it's okay to replicate this information for now if we need `isLast` often)